spread.yaml

```diff
@@ -167,6 +172,56 @@ debug-each: |
   journalctl -xe
 
 suites:
+ # Unit tests
+ tests/spread/unit/:
+   summary: unit tests for snapcraft
+   manual: true
+   systems:
+   - ubuntu-18*
+   prepare: |
+     apt-get update
+     apt-get install --yes \
+       git \
+       libapt-pkg-dev \
+       libffi-dev \
+       libnacl-dev \
+       libsodium-dev \
+       libssl-dev \
+       libxml2-dev \
+       libxslt-dev \
+       libyaml-dev \
+       python3-venv \
+       python3-yaml \
+       python3.6-dev \
+       rpm2cpio \
+       squashfs-tools \
+       subversion \
+       xdelta3
+     apt-mark auto \
```

Comment:
indent

Reply:
Seem to match indentation of the apt-get line, am I missing anything?